buildSrc/src/main/kotlin/Project.kt

```diff
@@ -71,7 +71,7 @@ object Versions {
   const val pluginAndroidGradle = "4.0.1"
   const val pluginKotlin = "1.4.0"
   const val pluginLicense = "0.8.5"
-  const val pluginDokka =  "1.4.32"
+  const val pluginDokka =  "1.4.10.2"
```

Comment:
BTW what's the issue with the updated Dokka? Is there a bug or structure is changed by intent? 
Anyway - would be nice to add a comment to not update to 1.4.32

Reply:
The file structure would be changed with 1.4.32 and we can't deploy it. For more detail please refer to the linked ticekt.

Reply:
Yeah, I checked that there is a breaking change in the dokka task that breaks the structure in [v1.4.30 ](https://github.com/Kotlin/dokka/releases/tag/v1.4.30). we do use `dokkaHtml` to generate the source sets which is changed to `dokkaHtmlPartial`. that might be the reason why 1.4.32 doesn't work properly.

Reply:
No, we will update it in the future, I have created to ticket for it.

Reply:
Ok then!